packages/babel-generator/test/fixtures/flow/internal-slot/output.js

```diff
@@ -0,0 +1,12 @@
+declare class C { static [[foo]]: T
```

Comment:
Nit: Can this go on a new line?

Reply:
Sure! The logic around inserting newlines wasn't immediately clear to me so I glossed over that part. Happy to make this change.